src/service/video-manager-impl.js

```diff
@@ -418,6 +426,78 @@ class VideoEntry {
   }
 
   /**
+   * Gets the provided metadata and fills in missing fields
+   * @return {!../video-interface.VideoMetaDef}
+   * @private
+   */
+  getMetaData_() {
+    let metaData = this.video.metaData;
+    if (!metaData) {
+      metaData = {};
+    }
+    if (!metaData.artist) {
+      metaData.artist = 'No artist';
+    }
+    if (!metaData.posterUrl) {
```

Comment:
(oh unless do fallback to AMP specific thumbnail in the schema.org metadata and then to the favicon)